### DISCUSSION TITLE
Add Xendit and Paymongo plugin payment for Fluxstore WooCommerce - Flutter E-commerce Full App

### DISCUSSION THREAD

**Discussion by gelocaindoy** [inspireui/support#5422] [Request new Native Payment]
Hey, Please I want you to include integrate Paystack native payment for xendit and paymongo payment intigration for your next update. 

BTW here is the link
https://www.xendit.co/en-ph/
https://www.paymongo.com/

**Comment by DevJac1** [comment]
Very nice and important updates. I'm also here in the Philippines. I think this would help regarding the payment integration.
in app.

**Comment by weareleadsgeno** [comment]
I think this is supported already.

**Reply by gelocaindoy** [reply]
hi @weareleadsgeno not yet sir.

**Comment by HanielQC** [comment]
Dear @gelocaindoy , @weareleadsgeno , @DevJac1 

Could you please share account so that we can develop Xendit payment into FluxStore apps?

Thank you and we appreciate your business.

**Reply by gelocaindoy** [reply]
Hi @HanielQC 

Noted. can i get the email address so i can add to my team to access the api.
Thanks!

**Comment by HanielQC** [comment]
https://youtu.be/11WXkUCmFKY - Xendit payment is supported by the latest FluxStore 3.10.0 and later. Thanks all for your support and cooperation.
